code/src/components/Summary.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react'
+
+import WelcomePage from './WelcomePage'
+import NameQuestion from './NameQuestion'
+import PreferenceQuestion from './PreferenceQuestion'
+import PastaKindQuestion from './PastaKindQuestion';
+import OtherIngredientQuestion from './OtherIngredientQuestion';
+import HungerQuestion from './HungerQuestion';
+import Overview from './Overview';
+
+const Summary = () => {
+  const [nameInput, setNameInput] = useState('')
+  const [preference, setPreference] = useState('')
+  const [sortOfPasta, setSortOfPasta] = useState('')
+  const [otherIngredient, setOtherIngredient] = useState('')
+  const [hungerInput, setHungerInput] = useState('')
+
+  const [step, setStep] = useState(1)
+
+  const onNameInputChange = (event) => {
+    setNameInput(event.target.value)
+  }
+  const onPreferenceChange = (event) => {
+    setPreference(event.target.value)
+  }
+  const onSortOfPastaChange = (event) => {
+    setSortOfPasta(event.target.value)
+  }
+  const onOtherIngredientChange = (event) => {
+    setOtherIngredient(event.target.value)
+  }
+  const onHungerInputChange = (event) => {
+    setHungerInput(event.target.value)
+  }
+  const onStepChange = () => {
+    setStep(step + 1)
+  }
+  const onPreviousQuestionChange = () => {
+    setStep(step - 1);
+  };
+
+  const onEnter = (event) => {
+    if (event.key === 'Enter') {
+      event.preventDefault();
+      setStep(step + 1);
+    }
+  };
```

Comment:
I dont think you need to have the if statement here for it to work.   This is how I did it and it works.

 ``` const onEnter = (event) => {  event.preventDefault();  setStep(step + 1); };```